scripts/ft-benchmark-data-sender.py

```diff
@@ -111,6 +111,11 @@ def commit_to_db(data: dict) -> None:
                         type=int,
                         required=True,
                         help='Number of users')
+    parser.add_argument('--user', type=str, default='unknown', help='User name')
```

Comment:
nit: I think `user` is a bit ambiguous here. `initiator` might be clearer and would be in line with the db column name. (In the postgres db naming a column `user` lead to lint errors, so instead it's named `initiator`.)

Reply:
Yep, it would be better to do this naming consistent with db scheme, but then I will need rename this across all files so probably will do it in another PR